docs/recipes/purescript.md

```diff
@@ -122,6 +122,9 @@ function maybe<A, B>(whenNone: () => B, whenSome: (a: A) => B, fa: Option<A>): B
 }
 ```
 
+Here TypeScript also provides exhaustiveness check because otherwise function
+ould lacks ending return statement and return type does not include 'undefined'.
```

Comment:
I believe a letter is missing here, maybe you wanted to write "would lack"?

Reply:
You are indeed right. Give me a sec, going to fix that commit…

Reply:
I believe it should be fixed as of now.